From 87d64613a9bce85df0f844bfec8c15f00ff73a66 Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Sun, 12 Jan 2025 13:47:06 +0200
Subject: [PATCH] [SW-0] hl-smi: info: rm check before get fw fit ver

Remove check of aip->is_linux_avail before calling
hlml_get_firmware_fit_version() the function is
checking it

Change-Id: Ied265de1defacdebf30ce7d88d3a14d8321f508c
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/app/hl-smi/info.cpp b/app/hl-smi/info.cpp
index e11e8c0..0b97eec 100644
--- a/app/hl-smi/info.cpp
+++ b/app/hl-smi/info.cpp
@@ -433,13 +433,11 @@
 	HL_SMI_LOG_INFO("\tAIP UUID\t\t\t: {}", info_field);
 	hl_smi_aip_log_status(aip);
 
-	if (aip->is_linux_avail) {
-		ret = hlml_get_firmware_fit_version(aip, fw_fit_version, sizeof(fw_fit_version));
-		if (ret == HLML_SUCCESS)
-			HL_SMI_LOG_INFO("\tFirmware [FIT] Version\t\t: {}", fw_fit_version);
-		else
-			HL_SMI_LOG_ERR("FAIL in function: hlml_get_firmware_fit_version");
-	}
+	ret = hlml_get_firmware_fit_version(aip, fw_fit_version, sizeof(fw_fit_version));
+	if (ret == HLML_SUCCESS)
+		HL_SMI_LOG_INFO("\tFirmware [FIT] Version\t\t: {}", fw_fit_version);
+	else
+		HL_SMI_LOG_ERR("FAIL in function: hlml_get_firmware_fit_version");
 
 	ret = hlml_get_firmware_spi_version(aip, fw_spi_version, sizeof(fw_spi_version));
 	if (ret == HLML_SUCCESS) {
